### PR TITLE
Brush up some bestiary automation for Howl of the Wild

### PR DESCRIPTION
**Pull request by simonward**
S, except for Stargut Hydra

### CODE REVIEW

File: packs/howl-of-the-wild-bestiary/sunscale-serpent.json [hunk 1]
```diff
@@ -288,7 +288,87 @@
...
+                        },
+                        "predicate": [
+                            "bond-with-mortal"
+                        ],
+                        "selector": "jaws-damage"
+                    },
+                    {
+                        "key": "DamageDice",
+                        "override": {
+                            "damageType": "slashing"
```
Comment (simonward) [inline packs/howl-of-the-wild-bestiary/sunscale-serpent.json:364]: Wasn't sure if this should be bludgeoning, but the ability says "typically piercing or slashing"

File: packs/howl-of-the-wild-bestiary/sky-fisher.json [hunk 2]
```diff
@@ -315,7 +315,7 @@
             "type": "lore"
         }
     ],
-    "name": "Skyfisher",
+    "name": "Sky Fisher",
```
Comment (simonward) [inline packs/howl-of-the-wild-bestiary/sky-fisher.json:318]: The creature name has a space, but the skyfisher vapors item does not.

File: packs/bestiary-effects/effect-grab-debris.json [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+                        "modifier": 6
+                    }
+                },
+                "key": "Strike",
+                "label": "PF2E.NPCAbility.Scroungefeather.GrabDebris.MagicBolt",
+                "predicate": [
+                    "self:effect:grab-debris:4"
+                ],
+                "range": {
+                    "max": 30
```
Comment (simonward) [inline packs/bestiary-effects/effect-grab-debris.json:103]: No range is listed, so I arbitrarily chose 30.

Comment (TikaelSol) [reply packs/bestiary-effects/effect-grab-debris.json:103]: It's 20 feet, as per the tossed scraps action

File: packs/bestiary-effects/effect-grab-debris.json [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+                },
+                "slug": "magic-bolt",
+                "traits": [
+                    "magical",
+                    "range-30"
+                ]
```
Comment (TikaelSol) [inline packs/bestiary-effects/effect-grab-debris.json:109]:
```suggestion
                    "max": 20
                },
                "slug": "magic-bolt",
                "traits": [
                    "magical",
                    "range-20"
                ]
```

**Pull request merged**